init.lua

```diff
@@ -17,6 +17,9 @@ vim.opt.rtp:append(home_dir .. "/.config/lvim/after")
 vim.cmd [[let &packpath = &runtimepath]]
 -- }}}
 
+_G.PLENARY_DEBUG = false -- Plenary destroys cache with this undocumented flag set to true by default
+require("impatient").enable_profile()
```

Comment:
You need to wrap this in a `pcall` so that it doesn't index nil when the plugin is not yet installed

Reply:
We do not since it is included with lvim, it does not use packer.